src/test/java/no/unit/nva/testutils/HandlerRequestBuilderTest.java

```diff
@@ -20,6 +21,8 @@
     public static final String PATH_PARAMETERS = "pathParameters";
     public static final String QUERY_PARAMETERS = "queryStringParameters";
     public static final String REQUEST_CONTEXT = "requestContext";
+    public static final String SOME_METHOD = "POST";
+    private static final String METHOD = "httpMethod";
```

Comment:
"method" as this is what is reflected in the code

Reply:
I fixed this in PR https://github.com/BIBSYSDEV/nva-testutils/pull/12, you will want to add travis CI, codacy to this project as the tests did not pass for the PR